docker/php-build-stepup/Dockerfile

```diff
@@ -20,19 +17,22 @@ ENV NVM_DIR /usr/local/nvm
 ENV NODE_VERSION 10
 
 # Install nvm with node and npm
-RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
-    && source $NVM_DIR/nvm.sh \
+RUN mkdir $NVM_DIR \
+    && curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.39.0/install.sh | bash \
```

Comment:
Is there no more generic way than a hardcoded version to download?

Reply:
I will look if they provide a `latest` or `stable` variety.

Reply:
Sadly there is no possibility to do this at this point.